saveJsonFile.py

```diff
@@ -0,0 +1,31 @@
+import googlemaps
+import json
+import requests
+
+API_KEY = 'AIzaSyDFtKtm0jnjckE-lB0LkO3SKlWmQIJush8';
+gmaps = googlemaps.Client(key=API_KEY)
+
+# wework civic : 37.778159, -122.411867
+gmaps_result = gmaps.places_radar(location=(37.778159, -122.411867), radius=500,
+                                 min_price=0, max_price=2, open_now=True, type='cafe')
```

Comment:
Indentation https://google.github.io/styleguide/pyguide.html?showone=Indentation#Indentation

Reply:
I'll refer to you!